edd-user-history.php

```diff
@@ -208,12 +208,12 @@ private function meets_requirements() {
 	 */
 	function maybe_update_plugin() {
 
-		require_once $this->directory_path . '/includes/updates.php';
 		// Bail early if not on an admin page
-		if ( ! is_admin() ) {
+		if ( ! is_admin() && ! ( defined( 'WP_CLI' ) && WP_CLI ) ) {
```

Comment:
I think this should be:

```suggestion
		if ( ! is_admin() && ! ( defined( 'WP_CLI' ) || ! WP_CLI ) ) {
```

Reply:
I think this still needs to be changed to `||` ? :thinking:

Reply:
I had it that way at one point and had to revert it to this. I need to test again but am stuck running a migration so I can't, but I think I need it this way because the `!` is checking both items at the same time. If `WP_CLI` is true I do need to run the code.

Reply:
I started a new migration this morning and checked. This is set up correctly to run in the admin and in `WP_CLI`.

Reply:
Oh I see. I was misreading before and thought the `!` was somewhere else. (Didn't realize these two were grouped in parenthesis together: `( defined( 'WP_CLI' ) && WP_CLI )` )